en/docs/quick-starts/attribute-config-sample.md

```diff
@@ -62,35 +42,23 @@ In the **Role** section, choose a role attribute from the application attributes
 
 3. You are now successfully logged in to the sample application. 
 
+4.  In the landing page of the application, you can see that the values of application attributes we mapped to the local attributes are shown instead of the local attributes. The application attribute, `Name` contains the value of the application role instead of the username since we configured it as the `role attribute URI`. 
```

Comment:
we mapped --> that were mapped

since we configured it --> since it was configured as

Reply:
try to avoid using "we" in docs